client/state/index.js

```diff
@@ -111,7 +112,9 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
+
+export const reducer = combineReducers( { ...extensions.reducers(), ...reducers } );
```

Comment:
Well, it was one or the other. I figured in the case of a name collision, it's better to not lose core functionality. Ideally, we'd namespace all extensions though, so namespacing issues like this can't happen.